site/en/docs/privacy-sandbox/feedback/index.md

```diff
@@ -228,6 +228,20 @@ groups** column of the tables in the [**Proposals** section](#proposals).
 
 {% Details %} {% DetailsSummary %}
 
+### Chrome-facilated testing
+
+**_We are seeking feedback on the [Chrome-faciliated testing plans](/docs/privacy-sandbox/chrome-testing/) along with tracking issues from initial third-party cookie deprecation._**
+
+{% endDetailsSummary %}
+
+If you rely on third-party cookie data for site functionality, we've launched a [public issue tracker](https://goo.gle/report-3pc-broken), where you can report site issues resulting from third-party cookie deprecation.
```

Comment:
so i can see why the public issue tracker bit is listed first but it does make clearly reading the sentence harder. Since it's just one sentence, and not a paragraph, maybe we should rewrite it to have the link at the end, that is:


If you rely on third-party cookie data for site functionality, you can now report site issues resulting from third-party cookie deprecation in the [public issue tracker](https://goo.gle/report-3pc-broken) .

Reply:
Updating.